gearsyncd/gearboxparser.cpp

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2019-2020 Broadcom Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gearboxparser.h"
+#include "phyparser.h"
+#include <vector>
+
+void 
+GearboxParser::notifyGearboxConfigDone(bool success)
+{
+    swss::ProducerStateTable *p = getProducerStateTable().get();
+
+    swss::FieldValueTuple finish_notice("success", std::to_string(success));
+    std::vector<swss::FieldValueTuple> attrs = { finish_notice };
+
+    p->set("GearboxConfigDone", attrs);
+}
+
+bool
+GearboxParser::parse()
```

Comment:
can you follow the coding convention of other orchagent code?

Reply:
@lguohan @jlevequeue I believe I responded to the comments regarding coding conventions, could you please review?